core/controllers/profile.py

```diff
@@ -251,7 +253,9 @@ class SignupPage(base.BaseHandler):
     def get(self):
         """Handles GET requests."""
         return_url = str(self.request.get('return_url', self.request.uri))
-
+        #validating return_url for no external redirection.
```

Comment:
Follow existing codebase standards: start with a capital letter, add space after '#'.

```
# Validate return_url to prevent redirection to external websites.
```

/cc @oppia/dev-workflow-team Can any of these be turned into automatic checks?

Reply:
There is a function for checking comments [here](https://github.com/oppia/oppia/blob/develop/scripts/pre_commit_linter.py#L1729). @nishealJ please update your branch with oppia/develop to run the lint tests on circle CI. Thanks